test/ttmlir/Dialect/TTIR/reshape/reshape_test.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline %s| FileCheck %s
```

Comment:
Since you are running it now with `--ttir-to-ttnn-backend-pipeline` just move it to the TTNN tests and rename it for the clarity, something like `Dialect/TTNN/reshape/reshape_folding_test.mlir`. Otherwise looks great!

Reply:
Moved the test as you mentioned, thanks for looking into this!